src/engine/enginebuffer.cpp

```diff
@@ -563,6 +574,18 @@ void EngineBuffer::slotTrackLoaded(TrackPointer pTrack,
     m_pTrackSampleRate->set(trackSampleRate.toDouble());
     m_pTrackLoaded->forceSet(1);
 
+    //  EveOSC begin
+    if (m_pConfig->getValue<bool>(ConfigKey("[OSC]", "OscEnabled"))) {
+        OscTrackLoadedInGroup(m_pConfig,
+                getGroup(),
+                pTrack->getArtist().toLatin1(),
+                pTrack->getTitle().toLatin1(),
+                (float)1,
```

Comment:
```suggestion
                1f,
```
we do not used C-Style casts in Mixxx

Reply:
Know all of a sudden I'm feeling old :-)